x/axelarnet/message_handler.go

```diff
@@ -120,12 +126,16 @@ func OnRecvMessage(ctx sdk.Context, k keeper.Keeper, ibcK keeper.IBCKeeper, n ty
 		Address: data.GetSender(),
 	}
 
+	rateLimitPacket := true
```

Comment:
I guess this works but feels so ad-hoc.

Reply:
yeah, not a fan, we don't have very consistent entry/exit points for transfers/messages through nexus yet